### PR TITLE
Wordpress boilerplate

### PR DESCRIPTION
**Pull request by foxyq**
set up Wordpress example with simple documentation

### CODE REVIEW

File: boilerplates/react-union-boilerplate-wordpress/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+```
+
+Build your app
+```sh
+    yarn build --proxy --release 
+```
+
+Copy built files from ```react-union-boilerplate-wordpress/build/public/assets/SampleApp/``` to your Wordpress installation ```wp-content/themes/ACTIVE_THEME_NAME/assets/js ```. This path should match with the public path we set in ```union.config.js```
+
+Update Wordpress file ```functions.php``` in your active theme directory to load React-Union scrips (in this case we put it to the end of ```function twentyseventeen_scripts()``` )
```
Comment (tommmyy) [inline boilerplates/react-union-boilerplate-wordpress/README.md]: scrips  -> scripts

**Comment by tommmyy** [comment]
@foxyq is there any progress?

File: boilerplates/react-union-boilerplate-wordpress/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        target: 'http://localhost:80/wpunion',
+	},    
+```
+
+Build your app
+```sh
+    yarn build --proxy --release 
+```
+
+Copy built files from ```react-union-boilerplate-wordpress/build/public/assets/SampleApp/``` to your Wordpress installation ```wp-content/themes/ACTIVE_THEME_NAME/assets/js ```. This path should match with the public path we set in ```union.config.js```
```
[the remark this answers is not in the document]
Comment (foxyq) [reply boilerplates/react-union-boilerplate-wordpress/README.md]: Automatic copying would be nice, but the script doesn't know the path to user's Wordpress installation (online, local - virtual machine, which folder etc.). That's why I kept this step manual. Is that ok?

**Comment by foxyq** [comment]
Changed the scripts mis-spelling, explained the build and manual copy instructions in comment.
Added (in progress) files for initializing CLI.

**Pull request closed without merging**